its/plugin/src/test/java/org/sonarsource/slang/ExternalReportTest.java

```diff
@@ -107,7 +107,7 @@ public void ktlint() {
   }
 
   private List<Issue> getExternalIssues(String componentKey) {
-    return newWsClient().issues().search(new SearchRequest().setComponentKeys(Collections.singletonList(componentKey)))
+    return newWsClient().issues().search(new SearchRequest().setProjects(Collections.singletonList(componentKey)))
```

Comment:
I needed to change this. To be discussed live.

Reply:
Discussed live, the change should be safe, as it doesn't change production code behavior.